### DISCUSSION TITLE
Calculate time BETWEEN checks instead of streaks

### DISCUSSION THREAD

**Discussion by temporarium** [iSoron/uhabits#1133] [Feature Requests]
Perhaps covered by other requests, but here goes...

I want to keep track of how often I get a haircut or replace a battery/filter/etc. I'll only be checking a day every few weeks/months, and would like the calculations to show an average time _between_ those 'habits' instead of the longest 'streak'.

Or am I missing something that's already implemented?

**Comment by hiqua** [comment]
I don't think that's directly visible from the UI, no. But I think it's not terribly useful for "normal" habits. Even for the cases you mention, and I have a few like these, I use them mostly as a reminder. If you really want other statistics about them, exporting as CSV and working on the data directly also works.

Is there a more convincing use case?

**Comment by temporarium** [comment]
Use-cases are up to the user. What I'm suggesting is very simple to implement. All the calculations are simply inversed. A flag in the habit definition is all that's required. 

Anyway, just a suggestion. Thanks for giving it some thought at least :-)
